### PR TITLE
Add embed cloudflare deployment and CI

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add embed deployment to cloudflare and CI.

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

https://embed.audius.co/playlist/DOPRl?flavor=card

**local**
npm run build:prod
npx wrangler preview --watch --env production

Viewable here:
https://embed.audius.co/embed/playlist/DOPRl?flavor=card

**ci**
run via this PR:
https://app.circleci.com/pipelines/github/AudiusProject/audius-client/16125/workflows/bab49d40-d813-4f08-815f-875030b59406

**client**

```
npx wrangler preview --watch --env production
```

Hitting https://example.com/embed/playlist/DOPRl?flavor=card yields the embed player!

image

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

File: .circleci/src/workflows/embed.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+when: << pipeline.parameters.run-embed-workflow >>
+jobs:
+  - embed-init
+  - embed-build-staging:
+      requires:
+        - embed-init
+      # filters:
+      #   branches:
```
Comment (raymondjacobson) [inline .circleci/src/workflows/embed.yml:8]: add back

File: .circleci/config.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ commands:
           name: Combine package-lock.json files to single file and also adds a random value at the end to cache bust if needed
           command: |
             ls -d -- packages/*/*-lock.json | awk -F packages '{printf "\"packages%s\" ", $2}' | xargs cat > << parameters.filename >>
-            echo cache-bust-v1 >> <<parameters.filename >>
+            echo cache-bust-1692933717156 >> <<parameters.filename >>
```
Comment (sliptype) [inline .circleci/config.yml:14]: Strange choice for the cache bust key but it's going away anyway

File: packages/embed/workers-site/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "version": "1.0.0",
```
Comment (sliptype) [inline packages/embed/workers-site/package.json:3]: Ah cool, no "name" field in here. That makes sense, I had to update the other workers sites names bc metro was complaining

**Review by sliptype — APPROVED** [review]
Yesss, good stuff

File: .circleci/config.yml (hunk 2 above)
Comment (raymondjacobson) [reply .circleci/config.yml:14]: oh ok, cool. yeah i needed to bust and just did Date.now(). v1 confused me

File: packages/embed/workers-site/package.json (hunk 3 above)
Comment (raymondjacobson) [reply packages/embed/workers-site/package.json:3]: TIL

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2991

File: .circleci/src/commands/@embed-commands.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    build-type:
+      default: 'prod'
+      type: string
+  steps:
+    - checkout
+    - attach_workspace:
+        at: ./
+    - run:
+        name: build
+        no_output_timeout: 60m
```
Comment (schottra) [inline .circleci/src/commands/@embed-commands.yml:15]: 😢

File: .circleci/src/jobs/@embed-jobs.yml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+          - node_modules
+          - packages/embed/node_modules
+          - packages/stems/node_modules
+          - packages/stems/dist
+
+
+embed-build-staging:
+  working_directory: ~/audius-client
+  docker:
+    - image: circleci/node:14.18
```
Comment (schottra) [inline .circleci/src/jobs/@embed-jobs.yml:39]: @sliptype Is this something for us to keep an eye on with the node upgrade?

File: packages/embed/README.md [hunk 6]
```diff
@@ -14,12 +14,3 @@ npm run start:stage
 # Run against production Audius services
 npm run start:prod
 ```
-
-## Deploying
```
Comment (schottra) [inline packages/embed/README.md:18]: Minor nit: Is it worth mentioning here that deployment happens through CI? Or is that just generally known?

File: packages/embed/workers-site/index.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    response.headers.set('X-Content-Type-Options', 'nosniff')
+    response.headers.set('X-Frame-Options', 'DENY')
+    response.headers.set('Referrer-Policy', 'unsafe-url')
+    response.headers.set('Feature-Policy', 'none')
+
+    return response
+  } catch (e) {
+    return new Response(e.message || e.toString(), { status: 500 })
+  }
+}
```
Comment (schottra) [inline packages/embed/workers-site/index.js:58]: I'm going to assume most of this content is copied over from the previous implementation?
I don't know enough about the headers we are setting to catch any issues with it

**Review by schottra — APPROVED** [review]

File: .circleci/src/jobs/@embed-jobs.yml (hunk 5 above)
Comment (sliptype) [reply .circleci/src/jobs/@embed-jobs.yml:39]: Yup, this is find for now. Will do a follow up pr to bring embed up to 18!

File: packages/embed/README.md (hunk 6 above)
Comment (raymondjacobson) [reply packages/embed/README.md:18]: i think just generally known, but i can add some notes

File: packages/embed/workers-site/index.js (hunk 7 above)
Comment (raymondjacobson) [reply packages/embed/workers-site/index.js:58]: yep, just defaults from cloudflare

**Pull request merged**